packages/amplify_core/lib/src/category/amplify_auth_category.dart

```diff
@@ -46,6 +48,68 @@ class AuthCategory extends AmplifyCategory<AuthPluginInterface> {
 
   /// {@template amplify_core.amplify_auth_category.sign_up}
   /// Create a new user with the given [username] and [password].
+  ///
+  /// <?code-excerpt "doc/lib/auth.dart" region="imports"?>
+  /// ```dart
+  /// import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
+  /// import 'package:amplify_flutter/amplify_flutter.dart';
+  /// ```
+  ///
+  /// <?code-excerpt "doc/lib/auth.dart" region="signup"?>
+  /// ```dart
+  /// Future<void> signUpUser({
```

Comment:
Might be worth calling out that this will vary based on the app's config.

```suggestion
  /// /// Signs a user up with a username, password, and email. The required 
  /// /// attributes may be different depending on your app's configuration.
  /// Future<void> signUpUser({
```